src/trusted/service_runtime/nacl_app_thread.c

```diff
@@ -169,7 +169,8 @@ struct NaClApp *NaClChildNapCtor(struct NaClApp *nap, int child_cage_id, enum Na
     if (!parent_nd) {
       continue;
     }
-
+    
+    
```

Comment:
nitpick but dont need the extra newline here

Reply:
Fixed.